torch/_torch_docs.py

```diff
@@ -1106,6 +1106,69 @@ def parse_kwargs(desc):
     tensor([ 0.0255, 0.1374])
 """)
 
+add_docstr(torch.diag_embed,
+           r"""
+diag_embed(input, offset=0, dim1=-2, dim2=-1) -> Tensor
+
+Creates a tensor with entries on a diagonal from :attr:`input` by
```

Comment:
This first sentence confuses me... Maybe `` Creates a tensor whose diagonals of certain 2D planes (specified by :attr:`dim1` and :attr:`dim2`) are filled by :attr:`input`. ``?